java-aws-lambda-maven/src/main/java/example/Handler.java

```diff
@@ -0,0 +1,79 @@
+package example;
+
+import com.amazonaws.services.lambda.runtime.Context;
+import com.amazonaws.services.lambda.runtime.RequestHandler;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Random;
+
+import com.rookout.rook.API;
+import com.rookout.rook.RookOptions;
+
+public class Handler implements RequestHandler<Map<String,String>, String> {
+    @Override
+    public String handleRequest(Map<String,String> event, Context context) {
+        RookOptions opts = new RookOptions();
```

Comment:
do we want to include the token here as well as an example?

Reply:
In the docs I wrote that you need to add the token as an environment variable. Do you think we should switch to token as a parameter instead? It's a bit less secure.